model/exception.yaml

```diff
@@ -1,33 +0,0 @@
-groups:
-  - id: exception
-    type: span
```

Comment:
also it was a type span before and now it's just attribute, can it be neglected?

Reply:
Before there was one for span, one for log.  AFAIK we're moving everything to attribute over time, just need to be careful not to do this to resource just yet.